src/notation/view/abstractnotationpaintview.cpp

```diff
@@ -715,35 +711,37 @@ PointF AbstractNotationPaintView::canvasCenter() const
     return toLogical(PointF(x, y));
 }
 
-std::pair<qreal, qreal> AbstractNotationPaintView::constraintCanvas(qreal dx, qreal dy) const
+std::pair<qreal, qreal> AbstractNotationPaintView::constraintCanvas(qreal x, qreal y, CoordinateSystem inputCoordinateSystem) const
```

Comment:
Now that this method also converts absolute to relative, let's rename it to `constrainedCanvasMoveDelta`, to clarify that it returns a delta/relative value.